<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="girangsnm" failures="10" tests="36" name="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" time="185.551" errors="2" timestamp="2022-09-17T20:11:15 WIB" skipped="0">
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="4.313"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="4.216"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="11.846">
    <failure message="expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.juaracoding.PageObjectTMS.Staff.TestLaporanKegiatanStaffTMS.staff_invalid_login1(TestLaporanKegiatanStaffTMS.java:70)
at ✽.Staff invalid loginA1(file:///C:/Users/girangsnm/git/Project-TMS/src/test/resources/features/Staff/01.LaporanKegiatanStaff.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="0.389">
    <failure message="expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.juaracoding.PageObjectTMS.Staff.TestLaporanKegiatanStaffTMS.staff_get_required_from_btn_choose_file(TestLaporanKegiatanStaffTMS.java:162)
at ✽.Staff get required from btn choose file(file:///C:/Users/girangsnm/git/Project-TMS/src/test/resources/features/Staff/01.LaporanKegiatanStaff.feature:64)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="5.412">
    <failure message="expected [Wrong username or password..!] but found [×
Wrong username or password..!]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Wrong username or password..!] but found [×
Wrong username or password..!]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.juaracoding.PageObjectTMS.Staff.TestLaporanKegiatanStaffTMS.staff_invalid_login3(TestLaporanKegiatanStaffTMS.java:84)
at ✽.Staff invalid loginA3(file:///C:/Users/girangsnm/git/Project-TMS/src/test/resources/features/Staff/01.LaporanKegiatanStaff.feature:22)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="4.200"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="4.223"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="4.127"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="4.257"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="0.357">
    <failure message="expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.juaracoding.PageObjectTMS.Staff.TestLaporanKegiatanStaffTMS.staff_get_required_from_text_area(TestLaporanKegiatanStaffTMS.java:423)
at ✽.Staff get required from text area(file:///C:/Users/girangsnm/git/Project-TMS/src/test/resources/features/Staff/01.LaporanKegiatanStaff.feature:177)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="4.193"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="0.070"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="4.206"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="10.590"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="4.201"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="4.179"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="0.129"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="0.092"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="4.203"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="2.146"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="4.194"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="2.476">
    <failure message="expected [false] but found [div]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [div]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.juaracoding.PageObjectTMS.Staff.TestLaporanKegiatanStaffTMS.staff_get_stack_name_shift(TestLaporanKegiatanStaffTMS.java:284)
at ✽.Staff get stack name shift(file:///C:/Users/girangsnm/git/Project-TMS/src/test/resources/features/Staff/01.LaporanKegiatanStaff.feature:115)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="4.255"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="40.141">
    <failure message="expected [false] but found [The filetype you are attempting to upload is not allowed.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [The filetype you are attempting to upload is not allowed.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:617)
at com.juaracoding.PageObjectTMS.Staff.TestLaporanKegiatanStaffTMS.get_warning_sentence(TestLaporanKegiatanStaffTMS.java:444)
at ✽.Get warning sentence(file:///C:/Users/girangsnm/git/Project-TMS/src/test/resources/features/Staff/01.LaporanKegiatanStaff.feature:188)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="0.110"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="4.195"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="5.182">
    <failure message="expected [flase] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [flase] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.juaracoding.PageObjectTMS.Staff.TestLaporanKegiatanStaffTMS.staff_invalid_login2(TestLaporanKegiatanStaffTMS.java:77)
at ✽.Staff invalid loginA2(file:///C:/Users/girangsnm/git/Project-TMS/src/test/resources/features/Staff/01.LaporanKegiatanStaff.feature:15)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="12.940"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="0.102"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="4.195"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="4.215"/>
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="2.384">
    <failure message="expected [false] but found [img]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [img]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.juaracoding.PageObjectTMS.Staff.TestLaporanKegiatanStaffTMS.image_not_shown(TestLaporanKegiatanStaffTMS.java:178)
at ✽.image not shown(file:///C:/Users/girangsnm/git/Project-TMS/src/test/resources/features/Staff/01.LaporanKegiatanStaff.feature:69)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="0.368">
    <failure message="expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.juaracoding.PageObjectTMS.Staff.TestLaporanKegiatanStaffTMS.staff_get_alert_from_select_box_tipe_shift(TestLaporanKegiatanStaffTMS.java:250)
at ✽.Staff get alert from select box tipe shift(file:///C:/Users/girangsnm/git/Project-TMS/src/test/resources/features/Staff/01.LaporanKegiatanStaff.feature:99)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="22.865">
    <error message="The step &#039;Staff clik&#039; is undefined.
You can implement this step using the snippet(s) below:

@When(&quot;Staff clik&quot;)
public void staff_clik() {
    // Write code here that turns the phrase above into concrete actions
    throw new io.cucumber.java.PendingException();
}
" type="io.cucumber.testng.UndefinedStepException">
      <![CDATA[io.cucumber.testng.UndefinedStepException: The step 'Staff clik' is undefined.
You can implement this step using the snippet(s) below:

@When("Staff clik")
public void staff_clik() {
    // Write code here that turns the phrase above into concrete actions
    throw new io.cucumber.java.PendingException();
}

at io.cucumber.core.runtime.TestCaseResultObserver.assertTestCasePassed(TestCaseResultObserver.java:69)
at io.cucumber.testng.TestCaseResultObserver.assertTestCasePassed(TestCaseResultObserver.java:23)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:133)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$5(CucumberExecutionContext.java:129)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:129)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at io.cucumber.testng.AbstractTestNGCucumberTests.runScenario(AbstractTestNGCucumberTests.java:35)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="0.227">
    <error message="The step &#039;Staff get required from select box divisi&#039; is undefined.
You can implement this step using the snippet(s) below:

@Then(&quot;Staff get required from select box divisi&quot;)
public void staff_get_required_from_select_box_divisi() {
    // Write code here that turns the phrase above into concrete actions
    throw new io.cucumber.java.PendingException();
}
" type="io.cucumber.testng.UndefinedStepException">
      <![CDATA[io.cucumber.testng.UndefinedStepException: The step 'Staff get required from select box divisi' is undefined.
You can implement this step using the snippet(s) below:

@Then("Staff get required from select box divisi")
public void staff_get_required_from_select_box_divisi() {
    // Write code here that turns the phrase above into concrete actions
    throw new io.cucumber.java.PendingException();
}

at io.cucumber.core.runtime.TestCaseResultObserver.assertTestCasePassed(TestCaseResultObserver.java:69)
at io.cucumber.testng.TestCaseResultObserver.assertTestCasePassed(TestCaseResultObserver.java:23)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:133)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$5(CucumberExecutionContext.java:129)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:129)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at io.cucumber.testng.AbstractTestNGCucumberTests.runScenario(AbstractTestNGCucumberTests.java:35)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff" name="runScenario" time="0.353">
    <failure message="expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.juaracoding.PageObjectTMS.Staff.TestLaporanKegiatanStaffTMS.staff_get_required_from_select_box_tipe_absen(TestLaporanKegiatanStaffTMS.java:388)
at ✽.Staff get required from select box tipe absen(file:///C:/Users/girangsnm/git/Project-TMS/src/test/resources/features/Staff/01.LaporanKegiatanStaff.feature:162)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- com.juaracoding.PageObjectTMS.Staff.TestRunnerTMSAStaff -->
